pilot/pkg/networking/core/v1alpha3/accesslog.go

```diff
@@ -329,7 +331,9 @@ func buildEnvoyFileAccessLogHelper(prov *meshconfig.MeshConfig_ExtensionProvider
 			fl.AccessLogFormat, needsFormatter = buildFileAccessJSONLogFormat(logFormat)
 		}
 	} else {
-		fl.AccessLogFormat, needsFormatter = buildFileAccessTextLogFormat("")
+		// Provider did not specify formatter. Fallback to Mesh Config. This is useful
+		// in using the default "envoy" provider which does not have formatter specified.
+		return fileAccessLogFromMeshConfig(p, mesh)
```

Comment:
should this only work for `envoy` provider?

Reply:
otherwise  we should update the desc
https://github.com/istio/api/blob/8a4564651730043c0600c112740952f65360a933/mesh/v1alpha1/config.proto#L842